core/rpc/src/state/error.rs

```diff
@@ -18,22 +18,27 @@ use error_chain::*;
 use client;
 use crate::rpc;
 use crate::errors;
+pub use internal::*;
 
-error_chain! {
-	foreign_links {
-		Client(client::error::Error) #[doc = "Client error"];
-	}
-
-	errors {
-		/// Provided block range couldn't be resolved to a list of blocks.
-		InvalidBlockRange(from: String, to: String, details: String) {
-			description("Invalid block range"),
-			display("Cannot resolve a block range ['{:?}' ... '{:?}]. {}", from, to, details),
+mod internal {
```

Comment:
Same here. This is so off-topic.

Reply:
I agree.  I was tired of seeing the warnings when compiling, but I can revert those changes.